com.unity.multiplayer.mlapi/Runtime/Messaging/MessageQueue/MessageQueueHistoryFrame.cs

```diff
@@ -16,23 +17,23 @@ public enum QueueFrameType
         }
 
         public bool IsDirty; //Used to determine if this queue history frame has been reset (cleaned) yet
-        public bool HasLoopbackData; //Used to determine if a dirt frame is dirty because rpcs are being looped back betwen HostClient and HostServer
+        public bool HasLoopbackData; //Used to determine if a dirt frame is dirty because messages are being looped back betwen HostClient and HostServer
         public uint TotalSize;
         public List<uint> QueueItemOffsets;
 
         public PooledNetworkBuffer QueueBuffer;
         public PooledNetworkWriter QueueWriter;
-        public RpcQueueHistoryFrame LoopbackHistoryFrame; //Temporary fix for Host mode loopback work around.
+        public MessageQueueHistoryFrame LoopbackHistoryFrame; //Temporary fix for Host mode loopback work around.
```

Comment:
Is the comment regarding the `temporary` nature of this fix still accurate? (the bane of comments is silent obsolescence - if we aren't sure if this comment is still reflecting reality - I'd remove it)

Reply:
I haven't changed anything about the loopback logic. I have no idea if there's still plans to refactor it or not... so I can't answer this question. :(

Reply:
This is still true until the default transport supports loopback.  The UNetTransport does not support this.
I have been told that UTPTransport will support this, and so when that happens we can remove that and clean up the server and client begin/end rpc methods in the NetworkBehaviour class a bunch.

Reply:
I think the RFC I'm working on right now is going to get rid of this, actually.